docs/source/en/glossary.mdx

```diff
@@ -282,21 +300,27 @@ A task that combines texts with another kind of inputs (for instance images).
 
 ## N
 
-### Natural language generation
+### Natural language generation (NLG)
 
 All tasks related to generating text (for instance talk with transformers, translation).
 
-### Natural language processing
+### Natural language processing (NLP)
 
 A generic way to say "deal with texts".
 
-### Natural language understanding
+### Natural language understanding (NLU)
 
 All tasks related to understanding what is in a text (for instance classifying the
 whole text, individual words).
 
 ## P
 
+### pipeline
+
+A pipeline is an abstraction referring to a series of steps that are executed in a specific order to process and transform data to be used by a downstream machine learning model for training or inference. Some example stages found in a pipeline might be data preprocessing, feature extraction, normalization and model training.
```

Comment:
Pipelines in the context of 🤗 Transformers don't involve training, so it'd probably be good to remove that to avoid any confusion.

```suggestion
A pipeline in 🤗 Transformers is an abstraction referring to a series of steps that are executed in a specific order to preprocess and transform data and return a prediction from a model. Some example stages found in a pipeline might be data preprocessing, feature extraction, and normalization.
```

Reply:
Oh yeah good catch!